docs/usage.md

```diff
@@ -4,61 +4,105 @@ This is lazy dog HOWTO to using CodeChecker analysis.
 It invokes Clang Static Analyzer and Clang-Tidy tools to analyze your code.
 
 ## Table of Contents
-* [Step 1: Integrate CodeChecker into your build system](#step-1)
-* [Step 2: Analyze your code](#step-2)
-    * [Cross-Compilation](#cross-compilation)
-    * [Incremental Analysis](#incremental-analysis)
-    * [Analysis Failures](#analysis-failures)
-    * [Avoiding or Suppressing False Positives](#false-positives)
-* [Step 3: View analysis results in command line or generate static HTML files](#step-3)
-* [Step 4: Store analysis results in a CodeChecker DB and visualize results](#step-4)
-* [Step 5: Fine tune Analysis configuration](#step-5)
-    * [Ignore modules from your analysis](#ignore-modules)
-    * [Enable/Disable Checkers](#enable-disable-checkers)
-    * [Configure Checkers](#configure-checkers)
-    * [Identify files that failed analysis](#identify-files)
-* [Step 6: Integrate CodeChecker into your CI loop](#step-6)
-    * [Storing & Updating runs](#storing-runs)
-    * [ Alternative 1 (RECOMMENDED): Store the results of each commit in the same run](#storing-results)
-        * [Jenkins Script](#storing-results-example)
-    * [Alternative 2: Store each analysis in a new run](#storing-new-runs)
-        * [Jenkins Script](#storing-new-runs-example)
-    * [Gerrit Integration](#gerrit)
-    * [Programmer checking new bugs in the code after local edit (and compare it to a central database)](#compare)
-    * [Setting up user authentication](#authentication)
-* [Updating CodeChecker to new version](#upgrade)
-* [Unique Report Identifier](#unique-report-identifier)
-* [Listing and Counting Reports](#listing-reports)
-    * [How reports are counted?](#how-report-are-counted)
-    * [Report Uniqueing](#report-uniqueing)
-    * [How diffs between runs are calculated?](#diffs-between-runs)
-
-## Step 1: Integrate CodeChecker into your build system <a name="step-1"></a>
-CodeChecker only analyzes what is also built by your build system.
-
-1. Select a module to build (open source tmux in this example).
-```sh
-cd tmux
-./configure
-```
-2. Clean that module. e.g. `make clean`
+- CodeChecker HOWTO
+  - Table of Contents
+  - Preface
+  - [Step 0:<a name="step-0"></a>](#step-0)
```

Comment:
Why are these anchors still here? I already wrote about this a couple of times. PLEASE REMOVE IT from the TOC from every places!

Reply:
Done.